### PR TITLE
Added some missing fields to some objects and some string constants

### PR DESCRIPTION
**Pull request by anorborg**

### CODE REVIEW

**Comment by jdconley** [comment]
:shipit:

**Comment by jdconley** [comment]
Have you tested this in the RealCrowd application, especially the web hooks we use for updating document statuses?

**Comment by anorborg** [comment]
Your question reminded be about some of the callback errors we get, which I think relate to the parsing. I made a Parser (just added) and just tested that in the RealCrowd application and everything seems to work as expected. I wanted to right some tests in the HelloSign project but couldn't find a straight forward way to start an temporary web server to act as a test Callback.

**Comment by jdconley** [comment]
Yeah, writing the integration tests there isn't straightforward.

You could test the parser using mocked HttpRequestMessage(s) with anonymized/mutated data from actual callbacks...

**Comment by anorborg** [comment]
K, added callback parser tests

File: src/RealCrowd.HelloSign/HelloSignCallbackParser.cs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            var p = new InMemoryMultipartFormDataStreamProvider();
+            await request.Content.ReadAsMultipartAsync(p);
+
+            var cb = HelloSignCallback.Parse(p.FormData["json"]);
+
+            // only validate the hash if the ApiKey has been set
+            if (string.IsNullOrEmpty(ApiKey))
+            {
+                var keyBytes = Encoding.ASCII.GetBytes(ApiKey);
+                var hmac = new System.Security.Cryptography.HMACSHA256(keyBytes);
```
Comment (jdconley) [inline src/RealCrowd.HelloSign/HelloSignCallbackParser.cs]: This is disposable, should toss it in a using()

File: src/RealCrowd.HelloSign/HelloSignUtilities.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+using System.Threading.Tasks;
+
+namespace RealCrowd.HelloSign
+{
+    public static class HelloSignUtilities
+    {
+        public static string GenerateEventHash(string apiKey, string eventTime, string eventType)
+        {
+            var keyBytes = Encoding.ASCII.GetBytes(apiKey);
+            var hmac = new System.Security.Cryptography.HMACSHA256(keyBytes);
```
Comment (jdconley) [inline src/RealCrowd.HelloSign/HelloSignUtilities.cs]: This is disposable. Need to dispose.

**Comment by jdconley** [comment]
Fix the hash algorithms that aren't being disposed and :shipit: :fireworks:

**Pull request merged**